SwiftReflector/GenericReferenceRenamer.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Dynamo;
+using SwiftReflector.SwiftXmlReflection;
+
+namespace SwiftReflector {
+	public class GenericReferenceRenamer {
+		BaseDeclaration context;
+		Func<int, int, string> genericNamer;
+		public GenericReferenceRenamer (BaseDeclaration context, Func<int, int, string> genericNamer)
+		{
+			this.context = Exceptions.ThrowOnNull (context, nameof (context));
+			this.genericNamer = Exceptions.ThrowOnNull (genericNamer, nameof (genericNamer));
+		}
+
+		public TypeSpec Rename (TypeSpec type)
+		{
+			bool changed;
+			var newType = Rename (type, out changed);
```

Comment:
Nit, but you can make this a one liner.
```suggestion
			var newType = Rename (type, out bool changed);
```

Reply:
TIL